### DISCUSSION TITLE
The email address I used with a now deleted account is not available for my new one

### DISCUSSION THREAD

**Discussion by sciencefictionspacecraft** [bookwyrm-social/bookwyrm#2919] [Q&A]
Hello.

I made a new account using a new email address and moved my info to it, because there is no way to change username.

Then I deleted the old account, figuring I would update my email address to the correct one.

But now Bookworm won't 'let go' of my old email address, which is the one I need to use, and I can't change it on the old account to release it because it has been deleted.

So, I am prevented from using my actual email address...!

**Comment by mouse-reeve** [comment]
Thank you, and apologies for the weirdness in how the app handles this case. There's a conversation on that here: https://github.com/bookwyrm-social/bookwyrm/issues/2911 -- I'm going to close this as a duplicate, since there's more context on the issue
